loader_hub/apify/dataset/base.py

```diff
@@ -0,0 +1,40 @@
+"""Apify dataset reader"""
+from typing import Any, Callable, Dict, List
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+
+class ApifyDataset(BaseReader):
+    """Apify Dataset reader. Reads data from Apify dataset.
+
+    Args:
+        apify_token (str): Apify token.
```

Comment:
```suggestion
        apify_token (str): Apify API token.
```

Reply:
I'm thinking we could rename this to `apify_api_token` to be precise? And consistent with langchain implementation

Reply:
yes please, and also the env var name. let's be as close to our LangChain implementation as possible, including the copy